pkg/api/types.go

```diff
@@ -12,6 +14,24 @@ const (
 	KubernetesNamespace = "io.k8s."
 )
 
+const (
+	// PullAlways means that we always attempt to pull the latest image.
+	PullAlways PullPolicy = "always"
+
+	// PullNever means that we never pull an image, but only use a local image.
+	PullNever PullPolicy = "never"
+
+	// PullIfNotPresent means that we pull if the image isn't present on disk.
+	PullIfNotPresent PullPolicy = "if-not-present"
+
+	// DefaultBuilderPullPolicy specifies the default pull policy to use
+	DefaultBuilderPullPolicy = PullIfNotPresent
+
+	// DefaultPreviousImagePullPolicy specifies policy for pulling the previously
+	// build Docker image when doing incremental build
+	DefaultPreviousImagePullPolicy = PullAlways
```

Comment:
1) how would you call it? --previous-pull-policy? ;-)
2) none if of the names I have in my mind are good
3) you don't want to confuse people (at least for now)